scripts/deployment/travis_before_deploy.sh

```diff
@@ -12,6 +12,33 @@ validate_env_var() {
     done
 }
 
+# ContentaCMS Files To Delete List Function
+# This function receives no argument
+# List of least minimum files needed for Contenta CMS to work
+contentacms_files_to_delete_list() {
+     list=(
+    '.circleci'
+    '.babelrc'
+    'docs'
```

Comment:
I agree with both. As docs keep growing and become more informative it would be good for the user to have a local reference, at the same time we have the online docs. 

I think that adding them would be a good idea, we should think when people don't have an internet connection, yet they have Contenta CMS and still want to read the docs. I know it is an unusual use case but it is still a possibility.